test/test_suites.py

```diff
@@ -54,13 +11,47 @@
 from test_suit.resilience_tests.test_kc_recreate import TestKcRecreate
 from test_suit.resilience_tests.test_kc_recreate_chaos import TestKcRecreateChaos
 from test_suit.resilience_tests.test_kc_restart import TestKcRestart
+from test_suit.test_auto_table_creation import TestAutoTableCreation
+from test_suit.test_auto_table_creation_topic2table import TestAutoTableCreationTopic2Table
+from test_suit.test_avro_avro import TestAvroAvro
+from test_suit.test_avrosr_avrosr import TestAvrosrAvrosr
+from test_suit.test_confluent_protobuf_protobuf import TestConfluentProtobufProtobuf
+from test_suit.test_json_json import TestJsonJson
+from test_suit.test_multiple_topic_to_one_table_snowpipe import TestMultipleTopicToOneTableSnowpipe
+from test_suit.test_multiple_topic_to_one_table_snowpipe_streaming import \
+    TestMultipleTopicToOneTableSnowpipeStreaming
+from test_suit.test_native_complex_smt import TestNativeComplexSmt
+from test_suit.test_native_string_avrosr import TestNativeStringAvrosr
+from test_suit.test_native_string_json_without_schema import TestNativeStringJsonWithoutSchema
+from test_suit.test_native_string_protobuf import TestNativeStringProtobuf
+from test_suit.test_schema_evolution_avro_sr import TestSchemaEvolutionAvroSR
+from test_suit.test_schema_evolution_drop_table import TestSchemaEvolutionDropTable
+from test_suit.test_schema_evolution_json import TestSchemaEvolutionJson
+# res tests
```

Comment:
nit: what is `res?`

Reply:
I added that for 'resilience tests', we can add the full word there. Can we move the comment back above the rest of the resilience tests?

Reply:
all the formatting is done automatically so I have no control, I simply remove it